.github/workflows/rust.yml

```diff
@@ -66,7 +68,34 @@ jobs:
       - name: generate kcov report
         shell: bash
         run: |
+
           for file in target/debug/deps/drop-*; do [ -x "${file}" ] || continue; mkdir -p "target/cov/$(basename $file)"; kcov --exclude-pattern=/.cargo,/usr/lib --verify "target/cov/$(basename $file)" "$file"; done
 
       - name: upload coverage
         uses: codecov/codecov-action@v1
+
+  deny:
+    name: cargo deny
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+        name: git checkout
+
+      - uses: actions-rs/toolchain@v1
+        name: install toolchain
+        with:
+          profile: minimal
+          toolchain: stable
+
```

Comment:
```suggestion

      - uses: Swatinem/rust-cache@v1
         with:
           key: "deny"

```

Reply:
Why remove the newline only between these two steps?

Reply:
Weirdly enough, it's GH which add this removal and readding of a line, it seems its diff only support showing at least a removal, not only adding lines.

Reply:
Doesn't show up in the actual diff for me